src/pages/Notebook.pug

```diff
@@ -93,10 +94,10 @@ block article
         We performed - again - a TdT tailing reaction with the NEB TdT and changed the AT-rich primer concentration to 20 nM, the incubation temperature to room temperature and 30 C° and the incubation times.
         After that, we isolated DNA from the produced gel to recover elongated ssDNA strands and measured their DNA concentration via the NanoDrop.
         
-        Moreover, we repeated the TdT tailing reaction with the ThermoFisher TdT and also changed the AT-rich primer concentration to 20 nM, the incubation temperature to room temperature and 37 C° and the incubation times.
-        Subsequently, we repeated the TdT tailing reaction with the ThermoFisher TdT again because we used the wrong buffer volume the day before.         
-        In another experiment, we investigated the TdT tailing reaction of samples that either contain or do not contain CoCl2 at room temperature and 37 °C, to see if the elongation occurred independently of the co-factor.
-        Then, we prepared a dNTP dilution of 100 µM and performed a TdT tailing reaction with the ThermoFisher TdT to compare the impact of the usage of dATP, dTTP, dGTP and dCTP on the reaction. 
+        Moreover, we repeated the TdT tailing reaction with the ThermoFisher TdT and also changed the AT-rich primer concentration to 20 nM, the incubation temperature to room temperature and 37 °C, and the incubation times.
+        Subsequently, we repeated the TdT tailing reaction with the ThermoFisher TdT because we used the wrong buffer volume the day before.         
```

Comment:
```suggestion
        Subsequently, we repeated the TdT tailing reaction with the ThermoFisher TdT again because we used the wrong buffer volume the day before.         
```

Reply:
das ist ja schon die zweite Wiederholung des Experimentes, deswegen würde ich das da stehen lassen